ex8.1_hierarchy.py

```diff
@@ -0,0 +1,87 @@
+class RegularUser:
+    """
+    managing the regular users
+    """
+    def __init__(self, username, password, manager=False):
+        self.username = username
+        self.password = password
+        self.manager = manager
+
+
+class SystemAdministrator:
+    """
+        managing the system administrator users
```

Comment:
Weird indentations, use pycharm's autoformat to fix your code - 
https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html

Reply:
You should look at the "files changed" section int the pull request to catch these kind of problems